examples/app-service/gateway-required-virtual-network-integration/README.md

```diff
@@ -0,0 +1,9 @@
+# Example: a windows App Service integrate with a gateway required virtual network
+
+This example provisions a windows App Service integrate with a gateway required virtual network.
+
+### Notes
+
+* create a virtual network gateway may takes about 40 min.
+* To configure a gateway required virtual network integration, please make sure the virtual network already have a gateway configured with point-to-site addresses.

```

Comment:
Can we not do this via terraform so the user doesn't have to?

Reply:
yes, user can use terraform to ceate a virtual network gateway configured with point-to-site addresses. in the example, I have shown that in the example tf files. 
In the notes, I want to mention the users how they can use the virtual network integration

Reply:
Could we do everything required here in the example? it shouldn't require the user doing any setup aside from maybe changing some variables